pkg/webui/console/components/location-form/index.js

```diff
@@ -135,7 +147,14 @@ class LocationForm extends Component {
   }
 
   render() {
-    const { initialValues, formTitle } = this.props
+    const {
+      initialValues,
+      formTitle,
+      validationSchema,
+      children,
+      locationFieldsDisabled,
+      allowDelete,
+    } = this.props
     const { error } = this.state
 
     const entryExists = hasLocationSet(initialValues)
```

Comment:
Do we want the notification when the location is updated automatically?
<img width="872" alt="Screenshot 2020-03-27 at 16 02 27" src="https://user-images.githubusercontent.com/16374166/77763939-833b2580-7044-11ea-800f-3467bd2362e6.png">

Reply:
Yes, the notification would go away as soon as the location has been set by the status update. I think otherwise it's good to notify that there is no location information yet.